sunpy/map/sources/iris.py

```diff
@@ -37,6 +37,9 @@ class SJIMap(GenericMap):
     """
 
     def __init__(self, data, header, **kwargs):
+        # Assume pixel units are arcesc if not given
```

Comment:
is this still needed with the change to `MapFactory`?

Reply:
Not sure; I'll double check all these statements I've added.